Containers/UserProfile/UserProfile.js

```diff
@@ -13,9 +13,23 @@ export default class UserProfile extends Component {
         }
     }
     render() {
+        const { profilePic } = this.props;
         return (
-            <View style={styles.container}>
-                <Icon color="#3c4859" name="portrait" type="badge" size={150} />
+            <View style={styles.viewContainer}>
+                {
+                    profilePic.length ?
+                        <TouchableOpacity style={styles.profilePictureContainer} onPress={() => this.props.takeProfilePic()}>
+                            <View style={styles.pictureContainer} >
+                                <Image
+                                    style={styles.profilePicture}
+                                    source={{ uri: profilePic }}
+                                />
+                            </View>
+                        </TouchableOpacity>
+                        : <TouchableOpacity style={styles.profilePictureContainer} onPress={() => this.props.takeProfilePic()}>
+                            <Icon color="#3c4859" name="portrait" type="badge" size={150} />
+                        </TouchableOpacity>
```

Comment:
logic to either display our default picture or the profile picture they have taken